src/olympia/activity/models.py

```diff
@@ -30,8 +30,8 @@ def is_expired(self):
         return self.use_count >= MAX_TOKEN_USE_COUNT
 
     def is_valid(self):
-        return (not self.is_expired() and
-                self.version.addon.latest_version == self.version)
+        latest_version = self.version.addon.find_latest_version()
```

Comment:
What channel should it use though, once we introduce channel to `latest_or_rejected_version` ? I left that out because I wasn't sure.